src/sysroot-fetcher.ts

```diff
@@ -36,8 +36,8 @@ export async function downloadLinuxSysroot(electronVersion: string, targetArch:
 
   d('writing sysroot to disk');
   const tmpTarFile = path.resolve(ELECTRON_GYP_DIR, `${electronVersion}-${fileName}`);
-  if (await fs.pathExists(tmpTarFile)) await fs.remove(tmpTarFile);
-  await fs.writeFile(tmpTarFile, sysrootBuffer);
+  if (fs.existsSync(tmpTarFile)) await fs.promises.rm(tmpTarFile, { recursive: true, force: true });
+  await fs.promises.writeFile(tmpTarFile, sysrootBuffer);
```

Comment:
Since this is a `graceful-fs` call, it'll need to be passed through `util.promisify`